theme/_variables.scss

```diff
@@ -3,4 +3,6 @@
 //Breadcrumb
 $breadcrumb-bg: #fff; // Inner background colour (Bootstrap override).
 
+$wb-so-background-color: #fff;
```

Comment:
Just for consistency with what was done with the breadcrumb override, maybe there'd be value in adding ``// Sign in / off`` above this line?